docs/telemetry.md

```diff
@@ -1,12 +1,111 @@
+
+## Metrics file
+
+Percona stores the Metrics file in one of the following directories on the local file system. The location depends on the product.
+
+* Telemetry root path - `/usr/local/percona/telemetry`
+
+* PSMDB root path -   `${telemetry root path}/psmdb/`
+
+* PS root path -   `${telemetry root path}/ps/`
+
+* PXC root path - `${telemetry root path}/pxc/`
+
+* PG root path - `${telemetry root path}/pg/`
+
+* Percona ToolKit - `${telemetry root path}/toolkit/`
+
+Percona archives the telemetry history in `${telemetry root path}/history/`.
+
+### Metrics file format
+
+The Metrics file uses the Javascript Object Notation (JSON) format, structured in key-value pairs with a colon separating them (:).
+
+* Keys are always strings enclosed in double quotes ("db_instance_id").
+
+* Values can be in the following data types:
+
+    * String (enclosed in double quotes)
+
+    * Number (integer or floating point)
+
+    * Booleans (true or false)
+
+    * Null (the absence of a value)
+
+    * Arrays (ordered collection of values)
+
+    * Objects (unordered collections of key-value pairs)
+
+Objects are enclosed in curly braces ({}), and arrays are enclosed in square brackets ([]). Whitespace is ignored, except when part of a string value.
+
+Percona reserves the right to extend the current set of JSON structure attributes in the future.  
+
+The top-level keys are the following:
+
+| Key name           | Requirement | Where used          | Description                                                                                              |
+|--------------------|-------------|---------------------|----------------------------------------------------------------------------------------------------------|
+| `db_instance_id`   | Mandatory   | DB plugin only     | This key is the ID of the componenticular DB instance. It is used for all telemetry results from the same database instance and is saved between instance restarts. |
+| `db_cluster_id`    | Optional    | DB plugin only     | This key is the ID of componenticular DB instances grouped as DB cluster. It can be taken from the database instance or generated and is used for all telemetry results within the same cluster. It is saved between database instance restarts. |
+| `db_replication_id`| Optional    | DB plugin only     | This key is the ID of a specific DB replication setup (Main→secondary). It can be taken from the database instance or generated and is used for all telemetry results within the same replication setup. It is saved between database instance restarts. |
+| `pillar_version`   | Mandatory   | DB plugin only     | This key identifies the exact version of Percona product. It is used for backward compatibility with the earlier Telemetry project. |
 
 An example of the data collected is the following:
```

Comment:
I would include only example, without above details about what is mandatory, what is not